app/src/main/java/org/wikipedia/settings/Prefs.kt

```diff
@@ -700,10 +699,6 @@ object Prefs {
         get() = PrefsIoUtil.getString(R.string.preference_key_places_wiki_code, WikipediaApp.instance.appOrSystemLanguageCode).orEmpty()
         set(value) = PrefsIoUtil.setString(R.string.preference_key_places_wiki_code, value)
 
-    var shouldShowOneTimePlacesSurvey
-        get() = PrefsIoUtil.getInt(R.string.preference_key_places_show_one_time_survey, PlacesFragment.SURVEY_NOT_INITIALIZED)
```

Comment:
`SURVEY_NOT_INITIALIZED` in `PlacesFragment` can be removed.

Reply:
done!